server/tasks/process_file.py

```diff
@@ -74,11 +70,17 @@ async def async_process_streaming_output(
             data = await consumer.getmany(timeout_ms=3000, max_records=batch_size)
             for tp, messages in data.items():
                 if messages:
-                    result_handler(messages)
+                    logger.info(f"Handling {messages=} in topic {tp.topic}")
```

Comment:
Seems like this (and the 2 logs below it) should probably be debug-level, I imagine we'll flood the prod logs with this at info-level